go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/pkg/profile v1.3.0
 	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/common v0.4.1
```

Comment:
Package not required. Should stop using this package in logs, see below in comments.

Reply:
Done.